packages/grafana-ui/src/components/Stack/Stack.internal.story.tsx

```diff
@@ -1,15 +1,17 @@
 import { Meta, StoryFn } from '@storybook/react';
 import React, { ReactNode } from 'react';
 
-import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+import { Alert } from '../Alert/Alert';
+import { Button } from '../Button';
+import { Card } from '../Card/Card';
+import { Text } from '../Text/Text';
 
 import { Stack } from './Stack';
```

Comment:
Why is the story only using Stack? What about HorizontalStack and VerticalStack?

Reply:
I'll change that, and I might remove vertical stack.